examples/vector_iteration.rs

```diff
@@ -1,14 +1,28 @@
+#[cfg(feature = "serde")]
+extern crate serde;
+
+#[cfg(not(feature = "serde"))]
 extern crate rustc_serialize;
 extern crate mustache;
 
 use std::str;
 use std::collections::HashMap;
 
-#[derive(RustcEncodable)]
+#[cfg_attr(not(feature = "serde"), derive(RustcEncodable))]
 struct User {
     name: String
 }
 
+#[cfg(feature = "serde")]
+impl serde::ser::Serialize for User {
```

Comment:
Ah of course, sorry I missed the `examples` path! I understand why you're doing it this way now.